src/results/existing_boiler.jl

```diff
@@ -18,7 +18,7 @@
 """
 function add_existing_boiler_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
     r = Dict{String, Any}()
-
+    r["size_mmbtu_per_hour"] = round(value(m[Symbol("dvSize"*_n)]["ExistingBoiler"]) / KWH_PER_MMBTU, digits=3)
```

Comment:
any reason not to add this to the markdown text with the results keys?

Reply:
No reason, added.